Package.swift

```diff
@@ -33,7 +33,7 @@ var awsCCommonPlatformExcludes = ["source/android",
                                   "source/linux/system_info.c",
                                   "bin/"] + excludesFromAll
 
-// includes arch/generic
+// includes arch/generic because the Swift PM doesn't like the necessary compiler flags.
```

Comment:
nit: Maybe refer to it as SwiftPM (which is how their docs call it). standalone PM has a lot of other connatations

Reply:
Thanks, updated.